ClientMapBasicTest.testMapStatistics_withClientOperations

```
java.lang.AssertionError: get latency
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.client.map.ClientMapBasicTest.testMapStatistics_withClientOperations(ClientMapBasicTest.java:1192)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK1.6-freestyle/32/testReport/junit/com.hazelcast.client.map/ClientMapBasicTest/testMapStatistics_withClientOperations/

*maintenance*
```
java.lang.AssertionError: get latency
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.client.map.ClientMapBasicTest.testMapStatistics_withClientOperations(ClientMapBasicTest.java:1192)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast%20Maintenance/job/Hazelcast-3.maintenance-OracleJDK8/com.hazelcast$hazelcast-client/500/testReport/junit/com.hazelcast.client.map/ClientMapBasicTest/testMapStatistics_withClientOperations/

*master*
```
java.lang.AssertionError: get latency
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at com.hazelcast.client.map.ClientMapBasicTest.testMapStatistics_withClientOperations(ClientMapBasicTest.java:1192)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.7/com.hazelcast$hazelcast-client/1288/testReport/junit/com.hazelcast.client.map/ClientMapBasicTest/testMapStatistics_withClientOperations/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java
@@ -1185,12 +1185,18 @@ public class ClientMapBasicTest extends HazelcastTestSupport {
             map.remove(i);
         }
 
-        assertEquals("put count", operationCount, stats1.getPutOperationCount() + stats2.getPutOperationCount());
-        assertEquals("get count", operationCount, stats1.getGetOperationCount() + stats2.getGetOperationCount());
-        assertEquals("remove count", operationCount, stats1.getRemoveOperationCount() + stats2.getRemoveOperationCount());
-        assertTrue("put latency", 0 < stats1.getTotalPutLatency() + stats2.getTotalPutLatency());
-        assertTrue("get latency", 0 < stats1.getTotalGetLatency() + stats2.getTotalGetLatency());
-        assertTrue("remove latency", 0 < stats1.getTotalRemoveLatency() + stats2.getTotalRemoveLatency());
+        assertEquals("put count: stats1" + stats1 + " stats2:" + stats2, operationCount,
+                stats1.getPutOperationCount() + stats2.getPutOperationCount());
+        assertEquals("get count : stats1" + stats1 + " stats2:" + stats2, operationCount,
+                stats1.getGetOperationCount() + stats2.getGetOperationCount());
+        assertEquals("remove count : stats1" + stats1 + " stats2:" + stats2, operationCount,
+                stats1.getRemoveOperationCount() + stats2.getRemoveOperationCount());
+        assertTrue("put latency : stats1" + stats1 + " stats2:" + stats2,
+                0 < stats1.getTotalPutLatency() + stats2.getTotalPutLatency());
+        assertTrue("get latency : stats1" + stats1 + " stats2:" + stats2,
+                0 < stats1.getTotalGetLatency() + stats2.getTotalGetLatency());
+        assertTrue("remove latency : stats1" + stats1 + " stats2:" + stats2,
+                0 < stats1.getTotalRemoveLatency() + stats2.getTotalRemoveLatency());
     }
 
     @Test(expected = UnsupportedOperationException.class)

